frontend/common/utils/utils.tsx

```diff
@@ -294,7 +266,7 @@ const Utils = Object.assign({}, require('./base/_utils'), {
         break
       }
       case 'AUTO_SEATS': {
-        valid = isScaleupOrGreater && Utils.getFlagsmithHasFeature('auto_seats')
```

Comment:
Appreciate this isn't really part of this PR but I'm not sure this logic is actually 100% correct? We don't allow enterprise SaaS customers to auto upgrade their seats (for commercial reasons). The code here: https://github.com/Flagsmith/flagsmith/blob/main/frontend/common/utils/utils.tsx#L289 seems to suggest that this would allow them to do so?

Reply:
Ah, ok yep that makes sense - will adjust as part of this 👍